run-clang-format.py

```diff
@@ -26,7 +26,7 @@
 print("Style: " + style)
 
 def runclangformat(filepath):
-    print("Checking: " + filepath)
+    # print("Checking: " + filepath)
```

Comment:
I say this because, we have 2 places that this runs.
* In our script
* In our CI here: https://github.com/veedstudio/render-node/blob/master/.github/actions/clang-format/action.yaml

So, sometimes we forget to update the latter.